openqaoa/problems/problem.py

```diff
@@ -583,6 +602,9 @@ def __init__(self, numbers=None):
         self.numbers = numbers
         self.n_numbers = None if numbers == None else len(self.numbers)
 
```

Comment:
Let's try to remove all the inconsistent spacing too

Reply:
I am not sure which spacing you are referring to here